sda-commons-shared-otel-agent/README.md

```diff
@@ -0,0 +1,6 @@
+# SDA Commons Shared OpenTelemetry agent
```

Comment:
Why did you choose "shared" over "server" as part of the module name?

Reply:
For the agent, it is just built as a standalone dependency that can be used for both servers and clients, if we consider also that some of the instrumented libraries are client dependencies.